radio/src/CMakeLists.txt

```diff
@@ -120,6 +120,9 @@ endif()
 
 if(ARCH STREQUAL ARM)
   include(targets/common/arm/CMakeLists.txt)
+  if(NOT CMAKE_CXX_COMPILER_VERSION VERSION_EQUAL "13.2.1")
+    message(FATAL_ERROR "Only arm gnu toolchain 13.2.rel1 is currently supported")
```

Comment:
I personally am probably of the same opinion as Raphael... let people be idiots and not use this toolchain... i.e. make it warning, but not fatal